tutorials/language-decl/index.en.html

```diff
@@ -36,9 +36,9 @@
 <!--TRANSLATORS must change -en in the line just above to the subtag for their language! -->
 <script src="../../javascript/doc-structure/article-2022.js"> </script>
 <script src="../../javascript/articletoc-2022.js"></script>
-<link rel="stylesheet" href="../../style/article-2022.css" />
-<link rel="stylesheet" href="language-decl-data/local.css" type="text/css" />
-<link rel="copyright" href="#copyright"/>
+<link rel="stylesheet" href="../../style/article-2022.css">
+<link rel="stylesheet" href="language-decl-data/local.css" type="text/css">
```

Comment:
We should also remove type="text/css" to avoid validator warnings.

Reply:
Thanks. Fixed now.